### PR TITLE
Updates to 3.74 Release Notes for patch release

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- Merge to `rhacs-docs`
- Cherry pick to `rhacs-docs-3.74`

[Issue](https://issues.redhat.com/browse/ROX-15803)

[Link to docs preview
](https://57283--docspreview.netlify.app/openshift-acs/latest/release_notes/374-release-notes.html)

QE review:
- [x] QE has approved this change.

Additional information:

[Errata](https://errata.engineering.redhat.com/docs/show/110896)

### CODE REVIEW

File: release_notes/374-release-notes.adoc [hunk 1]
```diff
@@ -330,13 +331,17 @@ If you are using the in-product docs, you can instead download the link:https://
 * The `v1/alerts/summary/counts` endpoint, which is used by the *Policy violations by severity* and *Policy violations by category* widgets on the Dashboard, was fixed to prevent Central from crashing in highly-scaled environments. (ROX-13829)
 * {product-title-short} previously was not able to correctly identify the signature of images that contained the same content but were stored in different registries, causing problems with alerts and enforcement. This issue has been fixed and {product-title-short} now correctly identifies signatures of images from different registries. (ROX-12400)
 
+[id="resolved-in-version-3741"]
+=== Resolved in version 3.74.1
+
+* Previously, Central could not connect to secured clusters when using a proxy, even if the proxy was specified in the proxy configuration file or in environment variables. This issue is fixed. (ROX-15279)
```
Comment (dhaus67) [inline release_notes/374-release-notes.adoc:337]: It's the other way around:
Sensor wasn't able to connect to Central's when using a proxy, central didn't have any issues with proxies.

**Review by dhaus67 — APPROVED** [review]
Just one comment regarding switching the proxy statement from `central -> proxy -> sensor`to `sensor -> proxy -> central`.

File: release_notes/374-release-notes.adoc [hunk 2]
```diff
@@ -330,13 +331,17 @@ If you are using the in-product docs, you can instead download the link:https://
 * The `v1/alerts/summary/counts` endpoint, which is used by the *Policy violations by severity* and *Policy violations by category* widgets on the Dashboard, was fixed to prevent Central from crashing in highly-scaled environments. (ROX-13829)
 * {product-title-short} previously was not able to correctly identify the signature of images that contained the same content but were stored in different registries, causing problems with alerts and enforcement. This issue has been fixed and {product-title-short} now correctly identifies signatures of images from different registries. (ROX-12400)
 
+[id="resolved-in-version-3741"]
+=== Resolved in version 3.74.1
+
+* Previously, Sensor on secured clusters could not connect to Central when using a proxy, even if the proxy was specified in the proxy configuration file or in environment variables. This issue is fixed. (ROX-15279)
```
Comment (opayne1) [inline release_notes/374-release-notes.adoc:337]: Just a suggestion, feel free to disregard some/all
```suggestion
* Previously, secured clusters on Sensor could not connect to Central when using a proxy, even if the proxy was specified in the proxy configuration file or in environment variables. This issue is fixed and Sensor on secured clusters now connects to Central when using a proxy with no issue. (ROX-15279)
```

**Review by opayne1 — COMMENTED** [review]
Just a small suggestion to wording. Feel free to disregard 😄

File: release_notes/374-release-notes.adoc (hunk 2 above)
Comment (kcarmichael08) [reply release_notes/374-release-notes.adoc:337]: Thanks - I don't think the first sentence is technically accurate but let me reword. Good suggestion for the second!

**Pull request merged**